### DISCUSSION TITLE
Cannot export const revalidate in server component

### DISCUSSION THREAD

**Discussion by anarkrypto** [vercel/next.js#49769] [App Router] [answered]
I'm having this error in appDir with **v13.4.2**

```tsx
'use server';

export const revalidate = 60;

export default async function Test () {
    return <></>
}
```
**Error:**
x Only async functions are allowed to be exported in a "use server" file.

Only occurs with the 'use client' directive.

But anyway caching is not happening.

Reproduce: https://codesandbox.io/p/sandbox/cannot-revalidate-server-component-dj8bin?file=%2Fapp%2Fpage.tsx%3A12%2C1

**Comment by fuma-nama** [comment, marked as answer]
The `"use server"` is used for [server actions](https://nextjs.org/docs/app/building-your-application/data-fetching/server-actions) only, it expects you to export an async function.

By default, Pages are considered as a server component, you don't have to specify `use server` at the top of the file

**Reply by Ra0R** [reply]
But why would adding "use server" lead to an error?
